### PR TITLE
Runs command

### PR DESCRIPTION
**Pull request by HyxonQz**
Runs command to check player dungeon floor completions. e.g !runs m7 hxqz

### CODE REVIEW

File: src/instance/minecraft/commands/RunsCommand.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    const floor = context.args[0]
+    const givenUsername = context.args[1] ?? context.username
+    let dungeonType = null
+
+    if (Object.keys(Catacombs).includes(floor)) {
+      dungeonType = 'catacombs'
+    }
+    if (Object.keys(MasterMode).includes(floor)) {
+      dungeonType = 'mastermode'
+    }
```
Comment (aidn3) [inline src/instance/minecraft/commands/RunsCommand.ts:38]: Check case-insensitive.

File: src/instance/minecraft/commands/RunsCommand.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    return `${givenUsername}: ${floor} - ${amount || 0}`
+  }
+} satisfies ChatCommandHandler
+
+async function getParsedProfile(hypixelApi: Client, uuid: string): Promise<HypixelSkyblockMemberRaw> {
+  return await hypixelApi
+    .getSkyblockProfiles(uuid, { raw: true })
+    .then((res) => res.profiles.filter((p) => p.selected)[0].members[uuid])
+}
```
Comment (aidn3) [inline src/instance/minecraft/commands/RunsCommand.ts:74]: Directly embed the code instead of extracting into a separate function.

**Review by aidn3 — CHANGES_REQUESTED** [review]

File: src/instance/minecraft/commands/RunsCommand.ts (hunk 1 above)
Comment (HyxonQz) [reply src/instance/minecraft/commands/RunsCommand.ts:37]: done

File: src/instance/minecraft/commands/RunsCommand.ts (hunk 2 above)
Comment (HyxonQz) [reply src/instance/minecraft/commands/RunsCommand.ts]: done

**Review by aidn3 — APPROVED** [review]

**Pull request merged**